packages/@react-aria/ssr/package.json

```diff
@@ -29,5 +29,16 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "engines": {
+    "node": ">= 12"
+  },
+  "targets": {
+    "main": {
+      "context": "node"
+    },
+    "module": {
+      "context": "node"
+    }
   }
```

Comment:
This is a hack to prevent parcel from inlining `NODE_ENV` during compile time. node 12 to ensure we don't end up with optional chaining in the built output, matching our other browser targeted code. I did a diff from what's currently published and there appear to be no other differences from my additions here.